src/Parser/Priest/Shadow/Modules/Features/Checklist.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+// import ITEMS from 'common/ITEMS';
+
+import SpellLink from 'common/SpellLink';
+import Wrapper from 'common/Wrapper';
+// import ItemLink from 'common/ItemLink';
+
+import CoreChecklist, { Rule, Requirement, GenericCastEfficiencyRequirement } from 'Parser/Core/Modules/Features/Checklist';
+import CastEfficiency from 'Parser/Core/Modules/CastEfficiency';
+import Combatants from 'Parser/Core/Modules/Combatants';
+// import ManaValues from 'Parser/Core/Modules/ManaValues';
+// import Velens from 'Parser/Core/Modules/Items/Velens';
+import LegendaryUpgradeChecker from 'Parser/Core/Modules/Items/LegendaryUpgradeChecker';
+import LegendaryCountChecker from 'Parser/Core/Modules/Items/LegendaryCountChecker';
+import PrePotion from 'Parser/Core/Modules/Items/PrePotion';
+
+import VoidTorrent from '../Spells/VoidTorrent';
+
+
+class Checklist extends CoreChecklist {
+  static dependencies = {
+    castEfficiency: CastEfficiency,
+    combatants: Combatants,
+
+    legendaryUpgradeChecker: LegendaryUpgradeChecker,
+    legendaryCountChecker: LegendaryCountChecker,
+    prePotion: PrePotion,
+    voidTorrent: VoidTorrent,
+  };
+
+  rules = [
+    new Rule({
+      name: 'Use core spells as often as possible',
+      description: <Wrapper>Spells such as <SpellLink id={SPELLS.VOID_BOLT.id} icon />, <SpellLink id={SPELLS.MIND_BLAST.id} icon /> and <SpellLink id={SPELLS.VOID_TORRENT.id} icon /> are your most important spells. Try to cast them as much as possible.</Wrapper>,
+      requirements: () => {
+        return [
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.VOID_BOLT,
+          }),
+          new GenericCastEfficiencyRequirement({
+            spell: SPELLS.MIND_BLAST,
+          }),
+        ];
+      },
+    }),
+    new Rule({
+      name: <Wrapper>Maximize <SpellLink id={SPELLS.VOID_TORRENT.id} icon />s</Wrapper>,
```

Comment:
How about "Maximize Void Torrent effectiveness"? Or "cast time"? (you can't really maximize a spell)

Reply:
Sounds good!